docs/frontend_doc.md

```diff
@@ -264,6 +264,10 @@ Return:
 
 ```
 
+`NodeAddr` MUST be URL encoded. For example `123.456.9:9000` MUST become
```

Comment:
I used `encodeURI()` to encode the entire endpoint, but it doesn't escape the ':' character ! Should I modify the endpoints and push the change on this branch ?